ERROR - 2021-12-06 20:38:19 --> Error connecting to the database: No connection could be made because the target machine actively refused it
CRITICAL - 2021-12-06 20:38:19 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
#0 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Users\freedcode\Documents\digisign-2.0\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(156): CodeIgniter\BaseModel->first()
#5 C:\Users\freedcode\Documents\digisign-2.0\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(20): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#6 C:\Users\freedcode\Documents\digisign-2.0\vendor\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#7 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\CodeIgniter.php(824): Myth\Auth\Controllers\AuthController->attemptLogin()
#8 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#9 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\freedcode\Documents\digisign-2.0\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\freedc...')
#12 {main}
ERROR - 2021-12-06 20:38:25 --> Error connecting to the database: No connection could be made because the target machine actively refused it
CRITICAL - 2021-12-06 20:38:25 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
#0 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1443): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Users\freedcode\Documents\digisign-2.0\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(156): CodeIgniter\BaseModel->first()
#5 C:\Users\freedcode\Documents\digisign-2.0\vendor\myth\auth\src\Authentication\LocalAuthenticator.php(20): Myth\Auth\Authentication\LocalAuthenticator->validate(Array, true)
#6 C:\Users\freedcode\Documents\digisign-2.0\vendor\myth\auth\src\Controllers\AuthController.php(87): Myth\Auth\Authentication\LocalAuthenticator->attempt(Array, false)
#7 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\CodeIgniter.php(824): Myth\Auth\Controllers\AuthController->attemptLogin()
#8 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController(Object(Myth\Auth\Controllers\AuthController))
#9 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\Users\freedcode\Documents\digisign-2.0\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\Users\freedcode\Documents\digisign-2.0\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\freedc...')
#12 {main}
